### ISSUE TITLE
compile error with musl libc

### ISSUE THREAD

**Issue by rofl0r** [ZerBea/hcxdumptool#171]
```
hcxdumptool.c: In function 'main':
hcxdumptool.c:8800:11: error: assignment of read-only variable 'stdout'
    stdout = fopen("/dev/null", "w");
           ^
make: *** [hcxdumptool] Error 1
```
according to C standard, stdout pointer is read only and may not be changed.

i suggest using `FILE* out` instead of stdout and assign it to stdout on startup.

**Comment by ZerBea** [comment]
Please give me some time to check it.

**Comment by n0w1re** [comment]
Ok, that was totally my fault. I already sorta fixed that issue in #168, but left the same construct at another place at it was. It was also pointless at that place ...

@rofl0r Please try compiling again with your setup with PR #174.

**Comment by ZerBea** [comment]
Merged, thanks.
